src/dcc/RailcomBroadcastDecoder.cxx

```diff
@@ -82,15 +85,15 @@ bool RailcomBroadcastDecoder::process_data(const uint8_t *data, unsigned size)
         {
             case dcc::RMOB_ADRLOW:
                 if (currentL_ == payload) {
-                    if (countL_ < REPEAT_COUNT) ++countL_;
```

Comment:
The brace style in this section (and below) is incorrect per the style guidelines.

Reply:
fixed